Source/Core/VideoCommon/CustomTextures.cpp

```diff
@@ -140,10 +140,11 @@ void HiresTexture::Update()
   }
 }
 
-void HiresTexture::Prefetch()
+void CustomTexture::Prefetch()
 {
   Common::SetCurrentThreadName("Prefetcher");
 
+  //TODO Fix this whole thing - Consensus reached at IRC logs MAY-2017
```

Comment:
Initially I just wanted to add a note for myself there, then I figured it would get transferred over, so that's what I ended up with, at the meeting of like 10 people it was concluded that I'll fix this thing as it's a minor thing and it's good for practise. Unless someone goes in and steals it away from me hehe. The calculation is broken and the intent of it is obsolete because it has some very old facts in mind. don't worry about it, I have the logs and I'll get on this one soon.